neva

```diff
@@ -53,6 +52,28 @@ local function file_exists(name)
     end
 end
 
+-- split a stringm a bit modified version of
+-- https://stackoverflow.com/a/20100401
```

Comment:
explain a little more  about this method? typo: stringm?

Reply:
It's a split function like in any other language, as Lua doesn't have one in its stdlib I had to make an implementation with some extra modifications (such as removing the trailing v from the versions) since it would only be used for that.

Also since the remote version list returns `stable` and `0.4.4` it's redundant to leave both since they are the same version, so `0.4.4` is removed from the table before being returned.

And yes, was a typo. Fixing it right now :slightly_smiling_face:

Reply:
Cool!